### PR TITLE
Feature/JA-58 User Controller

### PR DESCRIPTION
**Pull request by Zjyslav**
I added `Email`, `PasswordHash` and `DateCreated` properties to `User.cs`.
I added Admin as new `UserType`.
I created 'UserController' with all CRUD actions and corresponding views. You cannot set or edit `Id` and `DateCreated`.
Passwords provided by user are hashed before being sent to `DataAccess`.
`PasswordHash` is replaced with "***" before being sent to views.

Final user registration screens will have to be replaced or modified, at least not to let user to register as admin.

### CODE REVIEW

File: TutorLizard.BusinessLogic/Models/User.cs [hunk 1]
```diff
@@ -1,20 +1,44 @@
...
     public string Name { get; set; }
+
+    [Display(Name="Typ użytkownika")]
     public UserType UserType { get; set; }
 
-    public User(int id, string name, UserType type)
+    [Required(ErrorMessage = "Podaj adres email.")]
+    [EmailAddress(ErrorMessage = "Podaj prawidłowy adres email.")]
+    [Display(Name = "Adres email")]
+    public string Email { get; set; }
```
Comment (GrzegorzKmiecinski) [inline TutorLizard.BusinessLogic/Models/User.cs:21]: I'd add maxlength attribute just for safety

**Review by GrzegorzKmiecinski — APPROVED** [review]
looks good to me

File: TutorLizard.BusinessLogic/Models/User.cs (hunk 1 above)
Comment (Zjyslav) [reply TutorLizard.BusinessLogic/Models/User.cs:21]: I missed this comment before merging (I saw green and though it meant "Go!"). Should I put it on my fix branch, or leave it for later? I think we'll be adjusting attributes on all models anyway.

Comment (GrzegorzKmiecinski) [reply TutorLizard.BusinessLogic/Models/User.cs:21]: it was just a minor nitpick :) We can do this whenever it suits you

**Pull request merged**